bliss/conf/base_config.yaml

```diff
@@ -84,6 +84,9 @@ encoder:
         concat_psf_params: false
         log_transform: false
         rolling_z_score: true
+    data_augmentation:
+        do_data_augmentation: true
+        epoch_start: 1
```

Comment:
Do we need the epoch_start option?

Reply:
i.e., why not always do data augmentation beginning with epoch 0 if augmentation is enabled?

Reply:
It'd be best to remove `epoch_start` and to assume `epoch_start = 0`. It's easier on users to have few options to set, all else equal. Also, if we're using data augmentation, I think we'd want it from the beginning. Otherwise, we may overfit on some examples (the non-transformed ones) at first while underfitting on others.